packages/playground/resolve/index.html

```diff
@@ -107,6 +116,18 @@ <h2>resolve package that contains # in path</h2>
   import deepPath from 'resolve-exports-path/deep.json?url'
   text('.exports-deep-query', deepPath)
 
+  // deep import SVG w/ exports w/ query
+  import deepSvgPath from 'resolve-exports-path/deep.svg?url'
+  text('.exports-deep-svg-query', deepSvgPath)
+
+  // deep import HTML w/ exports w/ query
+  import deepHtmlPath from 'resolve-exports-path/deep.html?url'
+  text('.exports-deep-html-query', deepHtmlPath)
```

Comment:
Without any `exports` defined, this kind of thing normally works.

But when `exports` is added for the same file, then you start running into errors like:

```
[plugin:vite:import-analysis] Failed to parse source for import analysis because the content contains invalid JS syntax. You may need to install appropriate plugins to handle the .html file format.
C:/Users/MLM/Documents/GitHub/vite/packages/playground/resolve/exports-path/deep.html:4:19
2  |  <html>
3  |    <body>
4  |      <div>foo</div>
   |                    ^
5  |    </body>
6  |  </html>
    at formatError (C:\Users\MLM\Documents\GitHub\vite\packages\vite\src\node\server\pluginContainer.ts:315:31)
    at TransformContext.error (C:\Users\MLM\Documents\GitHub\vite\packages\vite\src\node\server\pluginContainer.ts:305:13)
    at TransformContext.transform (C:\Users\MLM\Documents\GitHub\vite\packages\vite\src\node\plugins\importAnalysis.ts:140:14)
    at Object.transform (C:\Users\MLM\Documents\GitHub\vite\packages\vite\src\node\server\pluginContainer.ts:583:20)
    at doTransform (C:\Users\MLM\Documents\GitHub\vite\packages\vite\src\node\server\transformRequest.ts:156:27
Click outside or fix the code to dismiss.
You can also disable this overlay by setting server.hmr.overlay to false in vite.config.js.
```


![](https://user-images.githubusercontent.com/558581/155840811-50023996-cde1-401e-8b1e-419fd2550c6e.png)